gemrb/core/PluginLoader.cpp

```diff
@@ -114,7 +104,7 @@ static bool FindFiles( char* path, std::list<char*> &files )
 	strcat( path, "*.dll" );
 	if (( hFile = ( long ) _findfirst( path, &c_file ) ) == -1L) {
 		//If there is no file matching our search
-		PrintDLError("Error looking up dlls: %s");
+    Log(ERROR, "PluginLoader", "Error looking up dlls.");
```

Comment:
Misaligned.

And probably now #include <string.h> is not needed any more.

Reply:
Oh, yep.